app/views/hostgroups/_form.html.erb

```diff
@@ -69,7 +69,6 @@
       <p/>
       <%= render "puppetclasses/classes_parameters", { :obj => @hostgroup } %>
       <p/>
-      <%= render "common_parameters/inherited_parameters" %>
       <h6><%= _('Host group parameters') %></h6>
```

Comment:
h4 looked way to big.
I don't think it's that bad that they are small, the tables are more important